packages/metrics/src/utils/captureError.ts

```diff
@@ -1,10 +1,7 @@
-import Raven from 'raven-js';
+import { captureException } from '@sentry/browser';
 import hasNewRelic from './hasNewRelic';
 
-export default function captureError(
-  error?: string | Event | Error | null,
-  options: Raven.RavenOptions = {},
-) {
+export default function captureError(error?: string | Event | Error | null) {
```

Comment:
Instead of making this breaking, we could take in extras still, and automatically add them in the scope.
https://git.musta.ch/airbnb/torch/blob/master/src/shared/metrics/torchLogger.ts#L45

Reply:
Good idea, added `withScope`.